### ISSUE TITLE
Allow debugging of client scripts

### ISSUE THREAD

**Issue by bkircher** [otris/vscode-janus-debug#44]
Currently, we only allow debugging of scripts that are executed on the server. In otris privacy, and possibly other projects, however scripts are also executed in the native JANUS client. Most of those script are executed in, e.g., a wizard or a dialog that is displayed to the user.

Maybe it is possible to debug those scripts too. Maybe we don't even have to change a lot to support that. Investigate.

**Comment by FabianTe** [comment]
Trying to understand this: Are you talking about for example the script that are executed in DOCUMENTS client context or script exit scripts?

**Comment by tineheller** [comment]
Scripts that are executed in DOCUMENTS client context, are executed in the browser and can be debugged with the debugger of the browser (it's not simple but possible). You cannot debug those scripts with this extension.
I'm not sure about otris privacy or other projects, but anyway, for now we focus rather on DOCUMENTS.

**Comment by FabianTe** [comment]
> Scripts that are executed in DOCUMENTS client context, are executed in the browser and can be debugged with the debugger of the browser

Yes. And I think this is totally fine. I wouldn't want to debug the client with the extension.

1. This extension focuses on the server scripting.
2. Debugging the client is mostly needed for those people working with Gentables or User exists. I assume that the people doing so, know what they are doing and are fine with debugging with the browsers dev tools.
3. Debugging the client is possible by attach the default chrome debugger that vscode has to the browser. This works fine.

I don't think debugging the client via this extension is usefull.

**Comment by bkircher** [comment]
> I'm not sure about otris privacy or other projects, but anyway, for now we focus rather on DOCUMENTS.

Yes. It was meant for other products like otris privacy.
